package.json

```diff
@@ -109,21 +109,22 @@
     "npm": "^9.0.0"
   },
   "devDependencies": {
-    "@babel/plugin-syntax-import-assertions": "^7.22.5",
-    "@babel/preset-typescript": "^7.22.5",
+    "@babel/preset-typescript": "^7.22.15",
     "@cypress/vue": "^6.0.0",
     "@fontsource/roboto": "^5.0.0",
     "@mdi/js": "^7.3.67",
     "@mdi/svg": "^7.0.96",
     "@nextcloud/babel-config": "^1.0.0",
     "@nextcloud/browserslist-config": "^3.0.0",
-    "@nextcloud/eslint-config": "github:nextcloud-libraries/eslint-config#fe67f6b84e3c13eb0ea2478f44939abc2fe7c99c",
+    "@nextcloud/eslint-config": "github:nextcloud-libraries/eslint-config#0d785eb6d1d62b91be1452759a615a476c87db4c",
     "@nextcloud/stylelint-config": "^2.3.1",
     "@nextcloud/vite-config": "github:nextcloud-libraries/nextcloud-vite-config#36ae8cf759a30e1b020d90a6c3585fa7b7734dc7",
     "@nextcloud/webpack-vue-config": "github:nextcloud/webpack-vue-config#1f4277c6ab14be63810f425ef6b762d8ca59697b",
+    "@vue/compiler-sfc": "^3.3.7",
```

Comment:
Do we really need to add this manually? The docs say, one doesn't:
https://vuejs.org/guide/scaling-up/tooling.html#vue-compiler-sfc

Installing it manually might break dependabot updates.